### DISCUSSION TITLE
Lost access to vercel account

### DISCUSSION THREAD

**Discussion by AgungAdityatama** [vercel/next.js#68439] [Help]
### Summary

I have another account on Vercel, but I forgot what email I used. I have a project link that is already connected/deployed on that account. Can you help me?

Note: I didn't use Git when deploying to Vercel because at that time it was my first time using Vercel and only followed tutorials on the internet that didn't use Git.

**Comment by Juneezee** [comment]
See https://vercel.com/guides/how-to-get-vercel-support.

You should send an email to [registration@vercel.com](mailto:registration@vercel.com).

**Reply by AgungAdityatama** [reply]
Thank you for giving me an answer. I will continue to contact the contact you provided.
